crates/starknet-devnet-server/src/api/http/endpoints/blocks.rs

```diff
@@ -44,3 +45,19 @@ pub(crate) async fn abort_blocks_impl(
 
     Ok(AbortedBlocks { aborted })
 }
+
+pub async fn update_gas(
```

Comment:
the method is added as an http endpoint, but not added to the router

Reply:
Added in 1f9b19dfad14f5062b15441b662f1bdc2a44e2d8 I thought that http endpoints will be deprecated soon

Reply:
But seriously though, why are we adding a new http endpoint? The last new endpoints were rpc-only, right?

Reply:
so remove or add? @marioiordanov @FabijanC ?

Reply:
remove

Reply:
HTTP endpoint was removed in af0793fa2d61f356e5e119d140064abbe42ec615